modules/lang-mustache/src/main/java/org/elasticsearch/script/mustache/MustacheScriptEngine.java

```diff
@@ -107,10 +117,25 @@ public String execute() {
             try {
                 template.execute(writer, params);
             } catch (Exception e) {
-                logger.error(() -> format("Error running %s", template), e);
+                if (logException(e)) {
```

Comment:
Nitpick: Rename method to something like `shouldLogException`

Reply:
:+1